Makefile

```diff
@@ -74,10 +70,10 @@ docker-image: clean-release _output/bin/dex
 proto: api/api.pb.go server/internal/types.pb.go
 
 api/api.pb.go: api/api.proto bin/protoc bin/protoc-gen-go
-	@protoc --go_out=plugins=grpc:. api/*.proto
+	@./bin/protoc --go_out=plugins=grpc:. --plugin=protoc-gen-go=./bin/protoc-gen-go api/*.proto
```

Comment:
I think it's OK either way. $(GOBIN) would be sufficient IMO since we'd be installing protoc relative to GOBIN either way so $(GOBIN)/protoc should always be safe. but, LGTM either way